### DISCUSSION TITLE
E3SM MPASO output  frequency and varibles

### DISCUSSION THREAD

**Discussion by jiaqii123** [E3SM-Project/E3SM#6464] [E3SM model help]
hello！

I need to check daily output of all ocean and ice varibles in my E3SM2.1 run. I don’t know how to change output frequency. I try to change STOP_OPTION to ndays but can’t get results(output files are below). So, what should I do?

Besides, I want to check diatom and small phytoplankton biomass in the sea ice but I can only check iceFluxphytoC in the run. What can I do to get those varibles? 

COMPSET: GMPAS
RESOLUTION: EC30to60E2r2

Thanks a lot!

![image](https://github.com/E3SM-Project/E3SM/assets/145311321/44071742-bab0-48ec-b29f-7929944cd051)

**Comment by rljacob** [comment]
Tagging @maltrud

**Reply by rljacob** [reply]
maybe @jonbob knows
